dist/templates/ovnkube-webhook.yaml.j2

```diff
@@ -43,3 +43,24 @@ spec:
     - port: 443
       targetPort: webhook-api
 ---
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: ovnkube-admission-webhook
+webhooks:
+  - name: ovnkube-webhook.ovn-kubernetes.svc
+    clientConfig:
+      service:
+        name: ovnkube-webhook
+        namespace: ovn-kubernetes
+        path: "/egressip"
+      caBundle: "{{ ca_pem_64 }}"
+    rules:
+      - operations: [ "CREATE" ]
+        apiGroups: ["k8s.ovn.org"]
+        apiVersions: ["v1"]
+        resources: ["egressip", "egressips", "eip"]
```

Comment:
whereas I believe you only want `"egressips"` here; the aliases are only for CLI purposes. They don't exist in the API.